test/test_common/environment.h

```diff
@@ -195,11 +195,15 @@ class TestEnvironment {
   jsonLoadFromString(const std::string& json,
                      Network::Address::IpVersion version = Network::Address::IpVersion::v4);
 
+// This function is not used for Envoy Mobile tests, and ::system() is not supported on iOS.
```

Comment:
Let's remove the part about this not being used for Envoy Mobile. The reason we're ifdef'ing it out on iOS is because `::system()` is not supported on iOS. So just:

```
// The `::system()` function is not available on iOS.
```